vignette/Validation.Rmd

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
This vignette is in the wrong directory, which is why it is not being built on CI (that would have errored in R CMD check due to the unstated dependencies). It should be in `vignettes` (with an s) not `vignette`. Normally I do

```
usethis::use_vignette("vignettename")
```

to do the initial setup as it is error-prone to do by hand.